core/templates/dev/head/directives.js

```diff
@@ -156,3 +156,17 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive('updateLanguage', ['$rootScope', function($rootScope) {
```

Comment:
Actually I think this should be put in i18n.js. That said, on second thoughts, should this really be a directive? It doesn't seem like a particularly reusable component to me -- it's a bit more like a global setting.

Two possible alternatives to consider: (a) adding this to the Base.js controller instead (i.e declare $scope.currentLang and have `<html ... lang=<[currentLang]> ...>`, (b) in the oppia.config in i18n.js, adding a command for directly getting the 'html' element in the DOM and setting the lang attribute.